lib/js/sportsday.js

```diff
@@ -221,22 +220,14 @@
       }
       $rootScope.pagesCount += 1;
       if ($rootScope.pagesCount >= 10) {
-        if (getCookie("stopSurveyAlert") === "false") {
+        if (getCookie("stopSurveyAlert") !== "true") {
```

Comment:
Use a `!==` comparison instead, to accomodate for the cookie simply not existing (in which case `getCookie("stopSurveyAlert")` just returns `undefined`)